src/parser.rs

```diff
@@ -0,0 +1,9 @@
+//! Higher level parsers and constants on top of tree-sitter primitives.
+
```

Comment:
[Optional] Let's add doc url, on where we are getting these symbol tag names. I think they are coming from vendored `node-types.json`, according to: https://tree-sitter.github.io/tree-sitter/using-parsers#static-node-types